phys/module_cu_ntiedtke.F

```diff
@@ -1039,14 +1035,10 @@ subroutine cumastrn  &
        if(ldcum(jl).and.ktype(jl).eq.1) then
            ikb = kcbot(jl)
            ikt = kctop(jl)
-           ztau = ztauc(jl) * scale_fac(jl)
+           ztau = ztauc(jl) * scale_tau(jl)
```

Comment:
Maybe it is better to set ztauc (360., 10800.), then ztau=ztauc*scale_fac

Reply:
@weiwangncar please check the code here. My suggestion is to add (360, 10800) to **ztauc**. There will be no additional time limit for ztau.

Reply:
@hichunxi I thought that is what I did based on your recommendation before. Here is the section in the new code:

           ztauc(jl) = max(ztmst,ztauc(jl))
           ztauc(jl) = max(360.,ztauc(jl))
           ztauc(jl) = min(10800.,ztauc(jl))
           ztau = ztauc(jl) * scale_fac(jl)

Reply:
@weiwangncar Sorry I missed it. It looks good now. I will approve this PR very soon.